CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Added
 - Add new tests for `issuer.name` including Internationalization settings.
-- A new test to ensure type urls are treated as a unordered set.
+- Add a new test to ensure type urls are treated as a unordered set.
+- Add new tests for `issuer.description` including Internationalization settings.
```

Comment:
```suggestion
- Add new tests for `issuer.description` that includes internationalization settings.
```

Reply:
I am not taking this suggestion as including is more active and precise than that includes.